docs/src/manual.md

```diff
@@ -161,7 +161,116 @@ ADD EXAMPLE.
 ## Adding new sets
 
 Dualization.jl can automatically dualize models with custom sets.
-To do this, the user needs to define the set and its dual set in MathOptInterface.jl and provide the functions
-`dual_set` and `set_dot` for the set the user wants to dualize.
+To do this, the user needs to define the set and its dual set and provide the functions:
 
-ADD EXAMPLE
+* `supported_constraint`
+* `dual_set` 
+
+If the custom set has some special scalar product, like `MOI.PositiveSemidefiniteConeTriangle` also needs
```

Comment:
I'd put the "like ..." in parentheses and link to the part of the moi docs that describes that cone and its dot product. Then the end of the sentence should say "... the user also needs to...".